### PR TITLE
Do some 🚮 in this repo

### PR DESCRIPTION
**Pull request by issyl0**
- I spotted some things that could do with 🚮.

### CODE REVIEW

**Comment by issyl0** [comment]
```
ERROR 2023/09/10 11:31:46 You have the value 'taxonomy' in the disabledKinds list. In Hugo 0.73.0 we fixed these to be what most people expect (taxonomy and term).
But this also means that your site configuration may not do what you expect. If it is correct, you can suppress this message by following the instructions below.
If you feel that this should not be logged as an ERROR, you can ignore it by adding this to your site config:
ignoreErrors = ["error-disable-taxonomy"]
```

**Comment by mattstratton** [comment]
why did you delete `contributing.json`?

File: config.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ dataDir = "data"
 archetypedir = "archetypes"
 PaginatePath = "blog"
 buildDrafts = false
-disableKinds = ["taxonomy", "taxonomyTerm"]
+disableKinds = ["taxonomy"]
```
Comment (mattstratton) [inline config.toml:12]: this change is throwing the error. while we can disable the error, what what the impetus of the change?

Comment (issyl0) [reply config.toml:12]: The fact that when I built the site locally a deprecation warning came up in my terminal about this term.

Comment (issyl0) [reply config.toml:12]: See also the commit message: https://github.com/devopsdays/devopsdays-web/commit/d849ca465e352c4add1ff6158a83d45903eb1e42. It seemed simple enough to fix. How I was wrong... 😆

**Comment by yvovandoorn** [comment]
@issyl0 is still relevant?

**Comment by issyl0** [comment]
If I back out the first commit about `taxonomyTerm` and keep the others, we can still get rid of some unnecessary things. I'll do this later today.

**Comment by issyl0** [comment]
@yvovandoorn @mattstratton Ready for review! Sorry for the delay.

**Pull request merged**